apps/web/pages/auth/logout.tsx

```diff
@@ -20,13 +20,27 @@ type Props = inferSSRProps<typeof getServerSideProps>;
 export function Logout(props: Props) {
   const { status } = useSession();
   if (status === "authenticated") signOut({ redirect: false });
+  const [emailChange, setEmailChange] = useState(false);
+  const [passReset, setPassReset] = useState(false);
   const router = useRouter();
   useEffect(() => {
     if (props.query?.survey === "true") {
       router.push(`${WEBSITE_URL}/cancellation`);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [props.query?.survey]);
+  useEffect(() => {
+    if (props.query?.emailChange === "true") {
+      setEmailChange(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.query?.emailChange]);
+  useEffect(() => {
+    if (props.query?.passReset === "true") {
+      setPassReset(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.query?.passReset]);
```

Comment:
why not simply use const isPassReset = router.query?. passReset ?

Reply:
agreed